### ISSUE TITLE
[VT-72] vit should support more than xterm

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#94]
_Ben Boeckel on 2012-11-26T20:18:37Z says:_ 

 At least screen-256color should be supported.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2012-11-26T20:18:37Z
Modified: 2014-07-21T22:07:24Z
```

**Comment by taskwarrior** [comment]
_steve rader on 2012-11-26T22:01:45Z says:_ 

 Agreed.

Vit is very much unlike Taskwarrior because it insists on a xterm terminal 
type.  I'll remove that restriction entirely.

I just ran Vit in screen with a few hacks and that appears to have uncovered
some bugs in colorization parsing.  I'll exterminate them, assuming doing so
isn't unreasonably difficult.  (And thus I'm leaving this feature request open.)

For the posterity, to get screen running Taskwarrior, I had to put the
following in my ~/.screenrc file...

<pre>
attrcolor b ".I"
termcapinfo xterm 'Co#256:AB=\E[48;5;%dm:AF=\E[38;5;%dm'
defbce "on"
</pre>

...as per http://www.frexx.de/xterm-256-notes/

**Comment by taskwarrior** [comment]
_steve rader on 2012-11-28T18:59:47Z says:_ 

 The "xterm restriction" has been removed as of Vit v0.4.

However, on my RHEL and Ubuntu machines, Vit does not look good when using
"screen" and Taskwarrior's default 256color configuration.

I have verified, by way of a test program, that the Curses libraries are
mangling the clean Taskwarrior xterm-256color colorization escape sequences 
into very complex ones, which the "screen" and "linux" terminal types do 
not handle properly.

I have hope (but have not not verified) that Vit with the "screen" or 
"linux" terminal types will look correctly when Taskwarrior is configured
to use 16color (ANSI colorization.)  If anyone tries this and I'm incorrect,
let me know.
